pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kubecmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	latestapi "github.com/openshift/origin/pkg/api/latest"
+	client "github.com/openshift/origin/pkg/cmd/client"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Rolls back a deploymentConfig to match a previous deployment.
+
+By default, only the pod template of the previous deployment is carried over for the
+rollback. Other aspects of the previous deployment, such as triggers and the previous
+strategy used must be explicitly selected for inclusion during the rollback.
+
+When the 'output' option is specified, the JSON of the deploymentConfig for the rollback
```

Comment:
If you would like to review the outcome of the rollback, pass '-o json' to print a JSON representation of the updated deployment configuration instead of executing the rollback.  This is useful if you're not quite sure what the outcome will be.